examples/pxScene2d/src/Makefile

```diff
@@ -457,7 +463,7 @@ pxscene-with-so-dfb: cleansceneobj $(PXSCENE_LIB_OBJS) $(LIBRTCORE_TARGET) $(LIB
 libpxscene.so:
 	rm -rf $(OBJDIR)
 libpxscene.so: CFLAGS_FULL = $(CFLAGS)
```

Comment:
Change -std=c++11 to $(EXTRA_COMPILER_FLAGS) and define the EXTRA_COMPILER_FLAGS in one of the above "ifeq ($(PX_PLATFORM),PX_PLATFORM_GENERIC_DFB)" checks as either:

EXTRA_COMPILER_FLAGS = -std=c++0x or EXTRA_COMPILER_FLAGS = -std=c++11